### ISSUE TITLE
popup to update obData from within OBV/flexBV

### ISSUE THREAD

**Issue by krikkiteer** [inflex/OpenBoardData#12]
it would be much quicker to update the obData files (and therefore contribute to a full-fletched knowledgebase) if there was a popup (maybe integrated in the annotations popup?) to update obData values.. (rather than remember the netname, search in obData file, enter values, reload boardview... asf)

**Comment by base64bits** [comment]
I completely agree, until this is possible this will probably not gain any traction.

**Comment by inflex** [comment]
All in good time.  I do agree with you, it's just that there's going to be a lot of underlying changes that need to be made, and features added before it can all bubble up to the top as a nice convenient update/edit facility.  

It's being worked on but right now there's some other more pressing matters that are occupying my time for the next couple of months.

**Comment by base64bits** [comment]
Completely understand, but I must say that even the current integration has helped me a ton, I have created step by step guides inside each board, now I just follow those steps and my work gets done a lot faster, this also makes teaching someone a lot easier. 

I will create a new issue with a suggestion I have for future upgrades.
